src/TuxAnimations.js

```diff
@@ -37,20 +25,32 @@ var makeTransition = function (displayName, transitions) {
       // componentDidEnter FUNCTION: use to define actions to run after an animation has entry completed
       // componentDidLeave FUNCTION: use to define actions to run after an animation has leave completed
   return React.createClass({
-    componentWillEnter: function (callback) {
-      setAnimationDomNode('enter', callback);
+    // setAnimationDomNode FUNCTION: function to handle manipulating and setting a TransitionGroup in the DOM
+      //@param action STRING: the transition key you are looking to affect
+      //@param callback FUNCTION: callback attribute passed in from the containing function to act on function completion
+    setAnimationDomNode: function (action, callback) {
+      var componentToAnimate = this.getDOMNode();
+      var startingAction = transitions[action];
+      var endingAction = transitions[action + 'Active'];
```

Comment:
Okay one last comment.  Can we change this to `+ '-active'`.  We will need to change our animation keys to `'enter-active'` and `'leave-active'` since object keys cant have hyphens in them unless they are in quotes but the hyphen lowercase syntax is actually the standard for css transitions so we should stick to it.